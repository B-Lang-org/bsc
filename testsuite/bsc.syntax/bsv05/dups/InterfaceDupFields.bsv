// This should not compile because
// the method name is repeated
interface Test #(parameter type a);
    method Action setInput(a x, a y);
    method Action setInput(a x, a y);
    method a getOutput();
endinterface: Test
