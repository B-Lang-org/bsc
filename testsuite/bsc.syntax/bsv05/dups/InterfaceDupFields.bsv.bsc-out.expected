checking package dependencies
compiling InterfaceDupFields.bsv
Error: "InterfaceDupFields.bsv", line 5, column 19: (P0073)
  Duplicate definition of method `setInput' in interface `Test'
